bin/node-sass

```diff
@@ -266,9 +266,18 @@ function watch(options, emitter) {
 
   gaze.on('changed', function(file) {
     var files = [file];
+
+    // descendents may be added, so we need a new graph
+    graph = buildGraph(options);
     graph.visitAncestors(file, function(parent) {
       files.push(parent);
     });
+
+    // Add children to watcher
+    graph.visitDescendents(file, function(child) {
+      gaze.add(child);
+      files.push(child);
```

Comment:
You don't want to compile the decedents of the changed file. Please remove this.